docs/checkit.js

```diff
@@ -732,7 +732,7 @@ CheckIt.prototype.displayCalendar = function(calendarObj) {
 };
 
 CheckIt.prototype.clearPage = function($div) {
-    // Remove all divs from page except #template
+    // Remove all calendar related divs from page
```

Comment:
this is still wrong. It removes the checkit related divs from inside `$div`. The rest are untouched.